src/components/CropImageModal/CropImageModal.test.tsx

```diff
@@ -28,6 +24,26 @@ jest.doMock('react-avatar-editor', () => ({
     }),
 }))
 
+class FakeImg {
+    onload: () => void
+    constructor(public width: number, public height: number) {
+        setTimeout(() => {
+            this.callOnLoad()
+        }, 0)
+    }
+
+    private async callOnLoad() {
+        if (this.onload) {
+            return this.onload()
+        }
+        await new Promise((resolve) => {
+            setTimeout(() => resolve, 10)
+        })
+
+        await this.callOnLoad()
```

Comment:
Won't this loop forevah? If the idea is to loop-wait for `this.onload` to be set then maybe a while loop is better?

```ts
private async callOnLoad() {
    while (!this.onload) {
        await new Promise((resolve) => void setTimeout(resolve, 10));
    }

    this.onload()
}
```

Reply:
It wasn't looping forever but your solution is cleaner so I'll use it :)

Reply:
Yep, second part of my comment (If blablabla) is where I realised what's all the fuss about. I know it wasn't.